api/src/services/users.js

```diff
@@ -93,10 +93,10 @@ const validateContact = (id, placeID) => {
   return db.medic.get(id)
     .then(doc => {
       if (doc.type !== 'person') {
-        return Promise.reject(error400('Wrong type, contact is not a person.'));
+        return Promise.reject(error400({ message: 'Wrong type, contact is not a person.', translationKey: 'contact.type.wrong'}));
```

Comment:
Why not use the new parameters (message, key, params) that you've added to `error400` function?